test/spmd/tensor/test_dtensor_ops.py

```diff
@@ -0,0 +1,875 @@
+    aten.special_airy_ai.default: {f32},
+    aten.special_bessel_j0.default: {f32},
+    aten.special_bessel_j1.default: {f32},
+    aten.special_bessel_y0.default: {f32},
+    aten.special_bessel_y1.default: {f32},
+    aten.special_chebyshev_polynomial_t.default: {f32},
+    aten.special_chebyshev_polynomial_u.default: {f32},
+    aten.special_entr.default: {f32},
+    aten.special_erfcx.default: {f32},
+    aten.special_hermite_polynomial_h.default: {f32},
+    aten.special_hermite_polynomial_he.default: {f32},
+    aten.special_i0e.default: {f32},
+    aten.special_i1.default: {f32},
+    aten.special_i1e.default: {f32},
+    aten.special_laguerre_polynomial_l.default: {f32},
+    aten.special_log_ndtr.default: {f32},
+    aten.special_modified_bessel_i0.default: {f32},
+    aten.special_modified_bessel_i1.default: {f32},
+    aten.special_modified_bessel_k0.default: {f32},
+    aten.special_modified_bessel_k1.default: {f32},
+    aten.special_ndtri.default: {f32},
+    aten.special_scaled_modified_bessel_k0.default: {f32},
+    aten.special_scaled_modified_bessel_k1.default: {f32},
+    aten.special_spherical_bessel_j0.default: {f32},
+    aten.special_xlog1py.default: {f32},
+    aten.special_zeta.default: {f32},
+    aten.split.Tensor: {f32},
+    aten.split_with_sizes.default: {f32},
+    aten.sqrt.default: {f32},
+    aten.sqrt_.default: {f32},
+    aten.squeeze.default: {f32},
+    aten.squeeze.dim: {f32},
+    aten.squeeze_.dim: {f32},
+    aten.stack.default: {f32},
+    aten.std.correction: {f32},
+    aten.std_mean.correction: {f32},
+    aten.sub.Scalar: {f32},
+    aten.sub.Tensor: {f32},
+    aten.sub_.Tensor: {f32},
+    aten.sum.IntList_out: {f32},
+    aten.symeig.default: {f32},
+    aten.take.default: {f32},
+    aten.tan.default: {f32},
+    aten.tanh.default: {f32},
+    aten.threshold.default: {f32},
+    aten.to_sparse.default: {f32},
+    aten.to_sparse.sparse_dim: {f32},
+    aten.topk.default: {f32},
+    aten.trace.default: {f32},
+    aten.transpose_.default: {f32},
+    aten.triangular_solve.default: {f32},
+    aten.tril.default: {f32},
+    aten.triu.default: {f32},
+    aten.trunc.default: {f32},
+    aten.unbind.int: {f32},
+    aten.unfold.default: {f32},
+    aten.unique_consecutive.default: {f32},
+    aten.unique_dim.default: {f32},
+    aten.unsqueeze_.default: {f32},
+    aten.upsample_bicubic2d.vec: {f32},
+    aten.upsample_bilinear2d.vec: {f32},
+    aten.upsample_linear1d.vec: {f32},
+    aten.upsample_nearest1d.vec: {f32},
+    aten.upsample_nearest2d.vec: {f32},
+    aten.upsample_nearest3d.vec: {f32},
+    aten.upsample_trilinear3d.vec: {f32},
+    aten.var.correction: {f32},
+    aten.var_mean.correction: {f32},
+    aten.vdot.default: {f32},
+    aten.view_as_complex.default: {f32},
+    aten.view_as_real.default: {f32},
+    aten.where.self: {f32},
+    aten.xlogy.Tensor: {f32},
+    aten.zero_.default: {f32},
+    aten.zeros.default: {f32},
+    aten.zeros_like.default: {f32},
```

Comment:
Out of curiosity, how did you generate this long list?

Reply:
use the `COLLECT_EXPECT` to generate, actually i did some small trick to do fake run of all ops and write to a file directly